blocks/api/parser.js

```diff
@@ -220,7 +220,8 @@ export function createBlockWithFallback( name, rawContent, attributes ) {
  */
 export function parseWithGrammar( content ) {
 	return grammarParse( content ).reduce( ( memo, blockNode ) => {
-		const { blockName, rawContent, attrs } = blockNode;
+		// eslint-disable-next-line no-unused-vars
```

Comment:
~If we're truly not using `innerBlocks`, what's the intent with disabling the rule?~

Noting your comment at https://github.com/WordPress/gutenberg/pull/2743#issuecomment-333920882 , I could go either way on whether it's sensible to leave as a placeholder. It should be fine.